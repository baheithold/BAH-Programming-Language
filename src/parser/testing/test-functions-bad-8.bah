# This file tests an invalid function

function sum(x, y) {
    return x + y;
}

var three = sum(37 - 36 2 ^ 1);
