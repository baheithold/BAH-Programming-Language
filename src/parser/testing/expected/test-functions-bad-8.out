ERROR, line 7: expected CPAREN, but encountered INTEGER_TYPE
